### PR TITLE
Remove primitives that return strings instead of functions

### PR DESCRIPTION
**Pull request by kmax12**
We were returning strings in some cases to improve performance. This breaks api with other primitives. We can simply return pandas objects instead add the string optimization into pandas backend.

### CODE REVIEW

File: featuretools/primitives/standard/aggregation_primitives.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ class Count(AggregationPrimitive):
     default_value = 0
 
     def get_function(self):
-        return 'count'
+        return pd.Series.count
```
Comment (kmax12) [inline featuretools/primitives/standard/aggregation_primitives.py]: this actually isn't as performant. added question on stackoverflow here: https://stackoverflow.com/questions/55731149/use-a-function-instead-of-string-in-pandas-groupby-agg

File: featuretools/primitives/standard/aggregation_primitives.py [hunk 2]
```diff
@@ -261,7 +261,7 @@ class Skew(AggregationPrimitive):
     stack_on_self = False
 
     def get_function(self):
-        return 'skew'
+        return pd.Series.skew
```
Comment (kmax12) [inline featuretools/primitives/standard/aggregation_primitives.py]: skew doesn't seem to matter as much as count

Screen Shot 2019-04-17 at 11 36 02 AM

File: featuretools/primitives/standard/aggregation_primitives.py [hunk 3]
```diff
@@ -26,7 +27,12 @@ class Count(AggregationPrimitive):
     default_value = 0
 
     def get_function(self):
-        return 'count'
+        # note: returning pd.Series.nunique errors for python2,
```
Comment (rwedge) [inline featuretools/primitives/standard/aggregation_primitives.py]: this note is about `Count`, so "pd.Series.count"?

File: featuretools/primitives/standard/aggregation_primitives.py [hunk 4]
```diff
@@ -261,7 +264,12 @@ class Skew(AggregationPrimitive):
     stack_on_self = False
 
     def get_function(self):
-        return 'skew'
+        # note: returning pd.Series.nunique errors for python2,
```
Comment (rwedge) [inline featuretools/primitives/standard/aggregation_primitives.py]: skew for this note

File: featuretools/primitives/standard/aggregation_primitives.py [hunk 5]
```diff
@@ -7,6 +7,7 @@
 
 from ..base.aggregation_primitive_base import AggregationPrimitive
 
+import featuretools as ft
```
Comment (rwedge) [inline featuretools/primitives/standard/aggregation_primitives.py]: can we import `is_python_2` instead of the entire featuretools module?

**Comment by rwedge** [comment]
Looks good

**Pull request merged**